### PR TITLE
Add binary sensor documentation for La Marzocco

### PR DESCRIPTION
**Pull request by zweckj**
## Proposed change

Add binary sensor documentation for La Marzocco

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase:  https://github.com/home-assistant/core/pull/108212/
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/lamarzocco.markdown [hunk 1]
```diff
@@ -44,6 +44,12 @@ Host:
 | Auto on/off | Allows to enable/disable the auto on/off schedule | all |
 | Steam boiler | Allows to enable/disable the steam boiler | all |
 
+## Binary Sensors
```
Comment (c0ffeeca7) [inline source/_integrations/lamarzocco.markdown]:
```suggestion
## Binary sensors
```

**Review by c0ffeeca7 — DISMISSED** [review]
Thank you, @zweckj :+1:

**Comment by zweckj** [comment]
> Thank you, @zweckj 👍

thanks for the fast reviews 💖 at some point I will get the capitalization correctly...

File: source/_integrations/lamarzocco.markdown [hunk 2]
```diff
@@ -44,6 +44,12 @@ Host:
 | Auto on/off | Allows to enable/disable the auto on/off schedule | all |
 | Steam boiler | Allows to enable/disable the steam boiler | all |
 
+## Binary sensors
+
+| Binary sensor name | Description | Available for machines | Remarks |
+|-------------|-------------| ---------------------- | ------- |
+| Water tank empty | Indicates whether the water tank needs a refill | all | - |
```
Comment (joostlek) [inline source/_integrations/lamarzocco.markdown]:
```suggestion
| Water tank empty | Indicates whether the water tank needs a refill. | all | - |
```

**Review by joostlek — DISMISSED** [review]

**Review by joostlek — APPROVED** [review]

**Pull request merged**